### ISSUE TITLE
[BUG] Slider component behaves different with iOS and Android

### ISSUE THREAD

**Issue by troelslenda** [kirbydesign/designsystem#3148]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
With introduction of the new slider, the experience is somewhat degraded on iOS devices.
When you interact with the slider component on Android devices the Y axis is locked expectedly, disabling page scroll momentarily. 
When viewing the same page on iOS, it seems very difficult to maintain a locked Y axis.  However it is possible if your interaction is very slow and precise.
 

## Describe how to reproduce the bug
1. Acquire an iOS device
2. Open Safari
3. Navigate to https://kirby.design
4. Select Slider
5. Interact with Basic Example Slider

## Which Kirby version was used?
 
8.9.0

## What was the expected behavior?
Do reproduction steps on an Android device to get the feel of it.

## Add any screenshots
N/A

## Please complete the following information:
- iPhone 13 mini: iOS 16.5.1
- Android A52: Android 13: Vivaldi 6.1.3035.110

## Are there any additional context?
N/A

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
